README.md

```diff
@@ -409,6 +409,7 @@ docker run --name gitlab -d \
     --volume /srv/docker/gitlab/gitlab:/home/git/data \
     sameersbn/gitlab:15.1.2
 ```
+IMAP [Microsoft Graph](https://docs.gitlab.com/ee/administration/incoming_email.html#microsoft-graph)
```

Comment:
I think it is nicer to have more bit detailed description.

````suggestion

With [Microsoft deprecating basic authentication](https://techcommunity.microsoft.com/t5/exchange-team-blog/basic-authentication-deprecation-in-exchange-online-may-2022/ba-p/3301866), you may want to configure IMAP using the Microsoft Graph API. See details on [GitLab documentation for "incoming email with Microsoft Graph"](https://docs.gitlab.com/ee/administration/incoming_email.html#microsoft-graph)

Here is a example command: 

```bash
docker run --name gitlab -d \
    --env 'GITLAB_INCOMING_EMAIL_ENABLED=true' --env `IMAP_USER=incoming@example.onmicrosoft.com' \
    --env 'GITLAB_INCOMING_EMAIL_ADDRESS=incoming+%{key}@example.onmicrosoft.com' \
    --env 'IMAP_METHOD=microsoft_graph' \
    --env 'IMAP_TENANT_ID=<YOUR-TENANT-ID>' --env 'IMAP_CLIENT_ID=<YOUR-CLIENT-ID>' \
    --env 'IMAP_CLIENT_SECRET=<YOUR-CLIENT-SECRET>'
```
````

This will be rendered like below;

With [Microsoft deprecating basic authentication](https://techcommunity.microsoft.com/t5/exchange-team-blog/basic-authentication-deprecation-in-exchange-online-may-2022/ba-p/3301866), you may want to configure IMAP using the Microsoft Graph API. See details on [GitLab documentation for "incoming email with Microsoft Graph"](https://docs.gitlab.com/ee/administration/incoming_email.html#microsoft-graph)
    
Here is a example command: 
    
```bash
docker run --name gitlab -d \
    --env 'GITLAB_INCOMING_EMAIL_ENABLED=true' --env `IMAP_USER=incoming@example.onmicrosoft.com' \
    --env 'GITLAB_INCOMING_EMAIL_ADDRESS=incoming+%{key}@example.onmicrosoft.com' \
    --env 'IMAP_METHOD=microsoft_graph' \
    --env 'IMAP_TENANT_ID=<YOUR-TENANT-ID>' --env 'IMAP_CLIENT_ID=<YOUR-CLIENT-ID>' \
    --env 'IMAP_CLIENT_SECRET=<YOUR-CLIENT-SECRET>'
```